src/Shared/Expenso.Shared.System.Serialization/Default/DefaultSerializerOptions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Text.Json;
+
+using Expenso.Shared.System.Serialization.Converters;
+using Expenso.Shared.System.Types.Messages;
+using Expenso.Shared.System.Types.Messages.Interfaces;
+
+namespace Expenso.Shared.System.Serialization.Default;
+
+public static class DefaultSerializerOptions
+{
+    public static readonly JsonSerializerOptions DefaultSettings = new()
+    {
+        Converters =
+        {
+            new InterfaceToConcreteTypeJsonConverter<IMessageContext, MessageContext>()
+        }
+    };
+}
```

Comment:
Verified. It isn't affecting whole application because it's an options which are passing as parameter Serialize() method. So a developer decide if use it or not.

Reply:
Agree